app/workers/deletes.php

```diff
@@ -470,16 +470,17 @@ protected function deleteFunction(Document $document, Document $project): void
 
         /**
          * Request executor to delete all deployment containers
+         * TODO: Re-enable. Disabled for now because of proxy. Container killed after inactivity automatically.
          */
-        Console::info("Requesting executor to delete all deployment containers for function " . $functionId);
-        $executor = new Executor(App::getEnv('_APP_EXECUTOR_HOST'));
-        foreach ($deploymentIds as $deploymentId) {
-            try {
-                $executor->deleteRuntime($projectId, $deploymentId);
-            } catch (Throwable $th) {
-                Console::error($th->getMessage());
-            }
-        }
+        // Console::info("Requesting executor to delete all deployment containers for function " . $functionId);
+        // $executor = new Executor(App::getEnv('_APP_EXECUTOR_HOST'));
+        // foreach ($deploymentIds as $deploymentId) {
+        //     try {
+        //         $executor->deleteRuntime($projectId, $deploymentId);
+        //     } catch (Throwable $th) {
+        //         Console::error($th->getMessage());
+        //     }
+        // }
```

Comment:
What do we plan to do here ? Eventually we want to merge this to master right ?

Reply:
This is not as important as it was before.
Executor now has a timer in which it deletes runtimes that didn't receive request for X time (10 mins default).

So if you delete your function, delete worker no longer needs to delete the runtime. Runtime will eventually get deleted by executor.

I keep it as TODO since it would be more optimal way to do it. But I would prefer to not have it for now since it would not work well with OPR Proxy.

I consider this a future optimization.

Reply:
Let's remove this code completely in that case. The executor will do this anyways 👍